ddl/table_split_test.go

```diff
@@ -56,32 +57,32 @@ func (s *testDDLTableSplitSuite) TestTableSplit(c *C) {
 	defer dom.Close()
 	atomic.StoreUint32(&ddl.EnableSplitTableRegion, 0)
 	infoSchema := dom.InfoSchema()
-	c.Assert(infoSchema, NotNil)
-	t, err := infoSchema.TableByName(model.NewCIStr("mysql"), model.NewCIStr("tidb"))
-	c.Assert(err, IsNil)
-	checkRegionStartWithTableID(c, t.Meta().ID, store.(kvStore))
+	require.NotNil(t, infoSchema, nil)
```

Comment:
this nil can be removed.

Reply:
Agree. Let me push a trivial patch for this.
```suggestion
	require.NotNil(t, infoSchema)
```